plugins/modules/ndo_mcp_global_policy.py

```diff
@@ -0,0 +1,341 @@
+    description:
+    - The amount of MCP packets that will be received before port disable loop protection action takes place.
+    - Defaults to 3 when unset during creation.
+    - The value must be between 0 and 255.
+    type: int
+    aliases: [ loop_factor, loop_detection_mult_factor ]
+  port_disable:
+    description:
+    - Enable or disable port disabling when MCP packets are recived.
+    - Defaults to C(enabled) when unset during creation.
+    type: str
+    choices: [ enabled, disabled ]
+    aliases: [ port_disable_protection ]
+  initial_delay_time:
+    description:
+    - The MCP initial delay time in seconds.
+    - Defaults to 180 when unset during creation.
+    - The value must be between 0 and 1800.
+    type: int
+    aliases: [ initial_delay ]
+  transmission_frequency_sec:
+    description:
+    - The MCP transmission frequency in seconds.
+    - Defaults to 2 when unset during creation.
+    - The value must be between 0 and 300.
+    type: int
+    aliases: [ tx_freq ]
+  transmission_frequency_msec:
+    description:
+    - The MCP transmission frequency in milliseconds.
+    - Defaults to 0 when unset during creation.
+    - The value must be between 0 and 999.
+    type: int
+    aliases: [ tx_freq_ms ]
+  state:
+    description:
+    - Use C(absent) for removing.
+    - Use C(query) for listing an object or multiple objects.
+    - Use C(present) for creating or updating.
+    type: str
+    choices: [ absent, query, present ]
+    default: query
+notes:
+- The O(template) must exist before using this module in your playbook.
+  Use M(cisco.mso.ndo_template) to create the Fabric Policy template.
+- Attempts to create any additional MCP Global Policies will only update the existing
+  object in the Fabric Policy template.
+seealso:
+- module: cisco.mso.ndo_template
+extends_documentation_fragment: cisco.mso.modules
+"""
+
+EXAMPLES = r"""
+- name: Create the MCP Global Policy object
+  cisco.mso.ndo_mcp_global_policy:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: fabric_template
+    name: mcp_global_policy_1
+    key: cisco
+    state: present
+  register: mcp_global_policy_1
+
+- name: Create the MCP Global Policy object with all attributes
+  cisco.mso.ndo_mcp_global_policy:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: fabric_template
+    name: mcp_global_policy_1
+    description: A Global MCP Policy
+    key: cisco
+    admin_state: enabled
+    per_vlan: enabled
+    loop_detection_factor: 3
+    port_disable: enabled
+    initial_delay_time: 180
+    transmission_frequency_sec: 2
+    transmission_frequency_msec: 10
+    state: present
+  register: mcp_global_policy_1
+
+- name: Update the MCP Global Policy object with UUID
+  cisco.mso.ndo_mcp_global_policy:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: fabric_template
+    name: mcp_global_policy_1
```

Comment:
-> name: mcp_global_policy_update
to show that we are updating the name

Reply:
Done